app/extensions/publications/endpoints/publications/versions/delete_version.py

```diff
@@ -0,0 +1,99 @@
+from datetime import datetime, timezone
+
+from fastapi import APIRouter, Depends, HTTPException
+from sqlalchemy.orm import Session
+
+from app.core.dependencies import depends_db
+from app.dynamic.config.models import Api, EndpointConfig
+from app.dynamic.endpoints.endpoint import Endpoint, EndpointResolver
+from app.dynamic.models_resolver import ModelsResolver
+from app.dynamic.utils.response import ResponseOK
+from app.extensions.publications.dependencies import depends_publication_version
+from app.extensions.publications.permissions import PublicationsPermissions
+from app.extensions.publications.tables.tables import PublicationVersionTable
+from app.extensions.users.db.tables import UsersTable
+from app.extensions.users.dependencies import depends_current_active_user_with_permission_curried
+
+
+class EndpointHandler:
+    def __init__(
+        self,
+        db: Session,
+        user: UsersTable,
+        version: PublicationVersionTable,
+    ):
+        self._db: Session = db
+        self._user: UsersTable = user
+        self._version: PublicationVersionTable = version
+
+    def handle(self) -> ResponseOK:
+        if self._version.Deleted_At is not None:
+            raise HTTPException(status_code=400, detail="Publication Version already deleted")
+
+        if self._version.Act_Packages:
+            raise HTTPException(status_code=400, detail="Publication Version has related Act Packages, cannot delete")
+
+        self._version.Deleted_At = datetime.now(tz=timezone.utc)
+        self._version.Modified_By_UUID = self._user.UUID
+        self._version.Modified_Date = datetime.now(tz=timezone.utc)
```

Comment:
Als je een datetime meer dan 1 keer gebruik, dan gebruiken we die _timepoint in de constructor. Zodat de dates altijd gelijk zijn